contents/blog/explore-lineage-api/index.mdx

```diff
@@ -0,0 +1,460 @@
+---
+title: Exploring Lineage History via the Marquez API
+date: 2021-06-25
+author: Michael Collado
+image: ./image.svg
+banner: ./banner.svg
+description: Explore lineage history via marquez
+---
+
+Managing a data pipeline means tracking changes. Sometimes changes to your code, sometimes changes to
+somebody else’s schema, sometimes to the contents of the data itself. Sometimes you need to
+trace the root cause of a problem- somebody changed an int to a string and all the downstream consumers
+broke. Sometimes you want to make a change and see how your consumers were affected- do all the jobs
+run significantly faster after you filter out “unused” records? Or did somebody [rely on those
+"unused" records](https://www.hyrumslaw.com/) to be present in the data? Do the recommendation models
```

Comment:
I advise splitting this paragraph somewhere in the middle - just to make it so that the first thing the reader encounters is approachable. Perhaps at "Do..." here?